docs/local.md

```diff
@@ -12,7 +12,7 @@
 
 ## Environment prerequisites
 
-- [Install **go 1.15**](https://golang.org/dl/)
+- [Install **go 1.16**](https://golang.org/dl/)
```

Comment:
We are requiring go 1.16 only for golic `go install`? Seems overkilling.
@kuritka can't we package golic as OCI image with `golic` entrypoint and execute it with docker?

btw working with Crossplane upstream - their machinery says that my golang 1.15 is incompatible with their support matrix ( they are up to 1.14 as of now). Just making a point that following specific go versions is apparently not the best developer experience, especially if you work over several projects.

Reply:
okay, it's also `controller-gen`, sigh. Let's stick with 1.16 as of now

Reply:
As I see it now: I checked kube builder issues and seem it will take some time until they switch. GO1.16 would be backward compatible. `go install` is extended but `go get` would work the same way as it did. 

To use lower than 1.16, would be nice to reuse [go-get-tool](https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile#L92) function from kube builder.